src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/DataAccess.scala

```diff
@@ -48,4 +48,36 @@ trait DataAccess
       workflowFailureQuery.schema ++
       workflowMessageQuery.schema ++
       workflowAuditStatusQuery.schema
+
+  def truncateAll: WriteAction[Int] = {
+    // important to keep the right order for referential integrity !
```

Comment:
What happens if multiple people are running tests concurrently while connected to the same database?